priv/mssql2012.sql

```diff
@@ -754,10 +754,11 @@ CREATE TABLE domain_events (
 CREATE INDEX i_domain_events_domain ON domain_events(domain);
 
 CREATE TABLE discovery_nodes (
-    node_name varchar(250),
     cluster_name varchar(250),
-    updated_timestamp BIGINT NOT NULL, -- in seconds
+    node_name varchar(250),
```

Comment:
General comment: I think that for all DB's there should be `NOT NULL` for these fields.